data/the emerald sword.txt

```diff
@@ -0,0 +1,803 @@
+			`	"Welcome back." They say.`
+			label question
+			`	"I hope you are ready to make your payment."`
+			branch cash
+				credits >= 12000000
+			choice
+				`	"I don't have that much money to spare."`
+					goto nomoney
+				`	"Are you ready to give me a discount?"`
+					goto haggle
+				`	"How about you tell me what I want and I don't blow up your ship and your pirate friends?"`
+					goto threat
+			label cash
+			choice
+				`	"I need to think about this. Give me some time."`
+					defer
+				`	"That's a lot of money just for information. Can you make it cheaper?"`
+					goto haggle
+				`	"How about you tell me what I want and I don't blow up your ship and your pirate friends?"`
+					goto threat
+				`	"Here's the money."`
+					goto pay
+			label nomoney
+			`	"I know you don't. I can arrange for the Payment to be made in different ways. Organ donation, prostitution, voluntarizing as a test subject... We'll need an appraisal first though."`
+			`	You politely decline.`
+			`	"Come back to me when you have it, then, and I shall provide you with the information. We have all the time in the universe. Until you die, or someone decides to destroy the cube, that is."`
+			`	They gesture, and the men promptly and diligently escort you back out to your ship.`
+				defer
+			label haggle
+			`	Mammon doesn't look pleased.`
+			`	"What part of non-negotiable did you not understand? You're pushing your luck, captain <last>."`
+				goto question
+			label threat
+			`	Mammon just smiles, not at all intimidated.`
+			`	"You think you can intimidate me? It's your funeral, human. If you were smart, you would know that my power far exceeds yours. You know I can't let you leave here alive now. It would set a bad precedent for me. Safety is my top priority."`
+			`	You look around, and the men in suit are pointing guns at you. Alien guns. Before you can react, Mammon shoots you from under the table, fatally injuring your abdomen. Your half conscious body is dragged out of the ship, and your fleet being blown out of the sky by alien weapons is the last thing you see.`
+				die
+			label pay
+			`	Mammon takes the credit chips, examining them.`
+			`	"Very good. The artifact you're looking for has been taken by some mercenaries. It seems to me that your friend messed with the wrong people. But that is not the information you paid for."`
+			`	They lift their hands to the holographic map of the galaxy in the ceiling. It zooms in, filling up the entire room with stars. In front of you, you recognize the border betwen the Deep and the Dirt Belt.`
+			`	"They have kept the artifact for themselves to sell on the Black market, and their base of operations is right here in Algieba."`
+			`	The hologram zooms further in, on Watcher, and they point at it.`
+			`	"You will find them in this uninhabited moon. They are hiding out in this forest, and the artifact is with them. Expect hostilities."`
+			`	You thank Mammon for their services and get up to leave.`
```

Comment:
In that mission, i used "they" as a genderless pronoun, since it seems to be what most people agree on. But yes, substituting some of them for Mammon is a good idea.